examples/grpc-server-with-prometheus/client/client.go

```diff
@@ -1,19 +1,39 @@
 package main
 
 import (
+	"bufio"
 	"fmt"
 	"log"
+	"net/http"
+	"os"
+	"strings"
+	"time"
 
 	"golang.org/x/net/context"
 	"google.golang.org/grpc"
 
+	"github.com/grpc-ecosystem/go-grpc-prometheus"
 	pb "github.com/grpc-ecosystem/go-grpc-prometheus/examples/grpc-server-with-prometheus/protobuf"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+var (
+	// Create some standard client metrics
+	grpcMetrics = grpc_prometheus.NewClientMetrics()
```

Comment:
let's move this right before line 30, having this here makes one feel like it should be there